v0/destinations/bqstream/transform.js

```diff
@@ -1,45 +1,17 @@
 /* eslint-disable no-console */
 const { EventType } = require("../../../constants");
-const {
-  defaultRequestConfig,
-  defaultPostRequestConfig,
-  CustomError,
-  getSuccessRespEvents,
-  getErrorRespEvents,
-  isOAuthDestination
-} = require("../../util");
+const { CustomError } = require("../../util");
 
-/**
- * This function puts the accessToken information in the transformed response
- * to facilitate for a successfully authorised event
- *
- * @param {*} event - The event on which transformation is being performed
- * @param {*} response - Transformation Response
- * @returns Transformation Response bound with token information(may not be required as such)
- */
-async function processAuth(event, response) {
-  // OAuth for BQStream destination
-  const { oauthAccessToken } = event.metadata;
-  if (!oauthAccessToken) {
-    throw new CustomError("Access token is empty/undefined", 500);
+const getInsertId = (properties, insertId) => {
+  if (
+    properties[insertId] &&
+    (typeof insertId === "string" || typeof insertId === "number")
+  ) {
+    return `${properties[insertId]}`;
   }
```

Comment:
I think we should still return `undefined` or `null` explicitly

Reply:
Doesn't make sense as no return statement in a function means the function returns `undefined`. But can make the change